DevelopGitHub/GitHubExtension/GitHubExtension.Security.DAL/Entities/SecurityRole.cs

```diff
@@ -0,0 +1,19 @@
+﻿
```

Comment:
Please remove unnecessary blank spaces. ReSharper can do that automatically through the all solution.